<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementInfoDialog</class>
 <widget class="QDialog" name="ElementInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Element Info</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>351</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelName">
      <property name="text">
       <string>Element Info: Hydrogen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelAbbreviation">
      <property name="text">
       <string>abbrev</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelAtomicNumber">
      <property name="text">
       <string>at#</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelAtomicMass">
      <property name="text">
       <string>mass</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelCharge">
      <property name="text">
       <string>charge</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ElementInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ElementInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
